### ISSUE TITLE
Content Structure Planning

### ISSUE THREAD

**Issue by awitherow** [Dcamo001/dcamo001.github.io#22]
# Questions

## What content items are needed for the site?

1. Home page of albums
2. gallery of photos within each album
3. blog post

# Structure Definitions

## Photo

- filename (need to figure out how strapi does file linking...)
- title
- description
- date photographed
- dimensions
- price

## Gallery of Photos

- title
- description
- array of multiple `Photo` objects

## Blog Post

**Comment by Dcamo001** [comment]
I'm not entirely sure a traditional purchase page is needed... I think it would be nice to be able to click on any image in the galleries and have a nice pop up overlay with options to put it in their cart and what size, etc, to buy a print

**Comment by awitherow** [comment]
Exactly... something similar to this: https://github.com/paylike/webshop-example (https://billigpoker.dk) where the popup has the add to cart functionality, then there is a cart checkout page.

so with that in mind, content structure for an image would be something like:

- filename (link to media, not sure how strapi handles this)
- title
- description
- photographer
- date photographed
- price

anything else?

**Comment by Dcamo001** [comment]
I think that covers it. Maybe resolution. and photographer might be redundant to have to list

**Comment by awitherow** [comment]
Nice, updated the main part of the post. I have another :45 or so that I can play around with `strapi` and see how it'll do for the future.

**Comment by Dcamo001** [comment]
Awesome. Definitely on the right track. How are you playing around with strapi? I don't really know how to get started

**Comment by awitherow** [comment]
@Dcamo001 get it set up and running locally and check this out -> https://strapi.io/documentation/concepts/concepts.html
https://strapi.io/documentation/getting-started/quick-start.html

**Comment by Dcamo001** [comment]
Got it. Went through the quick start to build my first API and set up a product content type. Do you think I should use bootstrap to set up a check out page? https://getbootstrap.com/docs/4.1/examples/checkout/
